### ISSUE TITLE
Add abcedary to vendor and source locally

### ISSUE THREAD

**Issue by joeltaylor** [codeschool/JavaScript.com#305]
The abcedary repo is private now so we need to source it locally.

**Comment by johndjameson** [comment]
Did this get addressed in https://github.com/codeschool/JavaScript.com/commit/97efd984f7c0e0de1ee11c3d1cf4308b518206ed?

**Comment by joeltaylor** [comment]
Yeah, that’s just a reminder for me when I do merge in the refactor.
